### DISCUSSION TITLE
Why port krohnkite instead of bismuth?

### DISCUSSION THREAD

**Discussion by piotr-dobrogost** [anametologin/krohnkite#43] [General]
It seems like https://github.com/Bismuth-Forge/bismuth was treated for some time and by many people as more polished continuation of Krohnkite thus I'm curious why the decision to port Krohnkite not Bismuth to Plasma 6?

**Comment by anametologin** [comment]
It wasn't a choice between, I was using Krohnkite until kwin6 and wanted to use it after.

**Reply by piotr-dobrogost** [reply]
Thanks for the answer.
Also out of curiosity; have you tried Bismuth and for some reason you did not like it or did you not see the reason to look for something else then Krohnkite?

**Comment by anametologin** [comment]
> have you tried Bismuth 

No

I didn't know that there is a fork of Krohnkite. I knew that other tile scrips existed but not try all of them.
